portstyle.css

```diff
@@ -7,32 +7,33 @@ body {
 	margin: 0;
 }
 
+.fontStyleHead {
```

Comment:
If you're going to apply this style to all your headers only it might be worth putting h1, h2, h3 and so on as the selector instead as it will save you typing class="fontStyleHead" every time you want to apply it.